### PR TITLE
Final Edits

### PR DESCRIPTION
**Pull request by ji-mmyliu**
Ken's slew of issues starting from [here](https://discord.com/channels/861455612093530162/861456555278073856/1109657907387183245)

### CODE REVIEW

File: src/components/blog/details/index.tsx [hunk 1]
```diff
@@ -45,12 +46,15 @@ export const BlogDetails = (): JSX.Element => {
                         <div className="card-authors-image">
                             <Link to={`/user/${post.author.username}`}><img className="circle" src={post.author.gravatar_url} /></Link>
                         </div>
-                        <div className="card-authors-text">
+                        <div className="card-authors-text" style={{ display: "flex", gap: "0.25em" }}>
```
Comment (ApocalypseCalculator) [inline src/components/blog/details/index.tsx:49]: hmm, maybe avoid inline styling for the sake of clarity

**Review by ApocalypseCalculator — APPROVED** [review]
LGTM

**Review by JasonLovesDoggo — APPROVED** [review]
LGTM, the only thing was what Patrick said...

MIGHT change org..leading to a list of short org objects. (like ann objects)

File: src/components/blog/details/index.tsx (hunk 1 above)
Comment (ji-mmyliu) [reply src/components/blog/details/index.tsx:49]: > hmm, maybe avoid inline styling for the sake of clarity

oh yep forgot to remove that thanks for noticing

**Pull request merged**